python/doc/bibliography.rst

```diff
@@ -243,4 +243,7 @@ Bibliography
     `pdf <http://alvand.basu.ac.ir/~dezfoulian/files/Numericals/Springer%20-%20Introduction%20to%20numerical%20analysis,%20Bulirsch,%20Stoer.%20(2ed.,%201993)(400dpi)(ISBN%200387978.pdf>`__
 .. [wand1994] Wand M.P, Jones M.C. *Kernel Smoothing*
     First Edition, Chapman & Hall, 1994.
+.. [Morio2015] Morio J., Balesdent M., *Estimation of Rare Event Probabilities in Complex Aerospace and Other Systems*,
```

Comment:
```suggestion
.. [Morio2015] Morio J., Balesdent M., *Estimation of Rare Event Probabilities in Complex Aerospace and Other Systems*,
```
This entry should be "morio2015" (lowercase "m"). Also, the list is organized in alphabetical order, so you need to move the entry up in the file.